### ISSUE TITLE
'Override Signature' required click location is nonintuitive

### ISSUE THREAD

**Issue by comex** [NationalSecurityAgency/ghidra#1589]
**Describe the bug**
Given this code in the decompiler:
```c
(**(code **)(lVar1 + 0x20))();
```
Suppose you want to override the signature of this indirect call.  Where should you right click to get "Override Signature" to appear in the menu?

The answer is between the first and second character of the line, the `(` and `*`.  Nowhere else will work.

This is nonintuitive, to say the least.  (When I was trying to figure this out, I initially tried right clicking between the parentheses, right clicking the call instruction in the disassembly, and right clicking after selecting the whole call, all to no avail.  I only found the correct click location by brute force.)

Note that:
- When calling a named function, you can get "Override Signature" by right clicking on the function name.
- You can sometimes get "Override Signature" for a call by right clicking one of its arguments, only if passing and/or computing that argument does not have a distinct instruction associated with it.

But in this case, the function is not named and there are no arguments.

**To Reproduce**

[Here is a simple test binary](https://github.com/NationalSecurityAgency/ghidra/files/4274571/ghidra-override-indirect.zip) that reproduces the issue.

**Expected behavior**

It would be nicer if you could get "Override Signature" by right clicking other nearby tokens, and/or by right clicking the call instruction in the disassembly view.

**Environment (please complete the following information):**
 - OS: macOS 10.15.4
 - Java Version: 11
 - Ghidra Version: master

**Comment by astrelsky** [comment]
I didn't even know indirect calls were overrideable.

**Comment by Jaguar83** [comment]
I can confirm a similar thing for the following code:
```
(*lpPcmkBuffer->lpBuffer->vTbl->Lock)(...)
```
I needed to click on the last "->" to get access to "Override Signature"

Version: 9.1.2

**Comment by dragonmacher** [comment]
Closed by 700e752
